src/client/app/saksbehandler/sokeboks/OppgaveModal.tsx

```diff
@@ -14,7 +16,8 @@ const åpneOppgave = (oppgave: SøkeboksOppgaveDto) => {
 };
 
 export function OppgaveModal(props: { oppgave: SøkeboksOppgaveDto; open: boolean; closeModal: () => void }) {
-	const { isFetched: isFetchedSaksbehandler, data: innloggetSaksbehandler } = useInnloggetSaksbehandler();
+	const innloggetSaksbehandler = useGlobalStateRestApiData<NavAnsatt>(RestApiGlobalStatePathsKeys.NAV_ANSATT);
```

Comment:
Skal vi ikke gå bort fra denne og heller bruke react-query? For react-query kan man sette staletime til Infinity for at det aldri skal refreshes.

Reply:
det har du rett i